scenarioo-server/src/main/java/org/scenarioo/rest/builds/BuildsImporterResource.java

```diff
@@ -98,6 +95,35 @@ public void reimportBuild(@PathParam("branchName") final String branchName,
 		ScenarioDocuBuildsManager.INSTANCE.reimportBuild(buildIdentifier);
 	}
 
+	@GET
+	@Path("importBuild/{branchName}/{buildName}/{comparisonBranchName}/{comparisonBuildName}/{comparisonName}")
+	@Produces({"application/json"})
+	public Response importBuild(
```

Comment:
oh - i see - that one looks like we need to discuss it further - I not like the REST service to wait for potentially endless unless some background tasks are done. We have to discuss a different solution here. @til77  wouldnt it be also okay for you if we simply provide another service that checks for a comparison result and returns nothing if it is not available yet and then let the build poll for that information until it is available?